compiler/luci/pass/src/ResolveCustomOpAddPass.cpp

```diff
@@ -70,6 +70,9 @@ int32_t get_broadcastTo_index_among_inputs_of(luci::CircleCustom *cop)
   return -1;
 }
 
+/// @note  When adding two input values of different shapes, Add will do the broadcasting.
+///        Therefore, Custom("BroadcastTo") is skipped,
+///        and the input, not the output, will be connected to y of Add.
```

Comment:
As I wrote in the previous comment, just removing `BroadcastTo` because of `Add` will do the broadcasting is not correct. We just skipped the the broadcasting to the input `Const` because of simplicity. 
```suggestion
// NOTE Broadcasting of input `Const` is skipped cause `Add` will do the broadcasting.
// TODO Implement broadcasting to the `Const` input.
```

Reply:
@seanshpark 

I think this makes it easier to clearly understand pass. 
This was reflected in this PR. Thank you a lot! :)